Sources/UnleashProxyClientSwift/Client/UnleashProxyClientSwift.swift

```diff
@@ -62,7 +62,7 @@ public class UnleashClientBase {
             self.metrics = Metrics(appName: appName, metricsInterval: Double(metricsInterval), clock: { return Date() }, disableMetrics: disableMetrics, poster: urlSessionPoster, url: url, clientKey: clientKey, customHeaders: customHeaders, connectionId: connectionId)
         }
         
-        self.context = Context(appName: appName, environment: environment)
+        self.context = Context(appName: appName, environment: environment, sessionId: String(Int.random(in: 0..<1_000_000_000)))
```

Comment:
QQ is calculateContext overriding the sessionId with the providedContext session id (if present)?

Reply:
yes